django_netjsonconfig/api/views.py

```diff
@@ -1,9 +1,15 @@
 from ..models import Template
-from .generics import BaseSearchTemplate
+from .generics import BaseGetTemplateView, BaseSearchTemplate
+
+
+class SharedTemplate(BaseGetTemplateView):
+    template_model = Template
+    queryset = Template.objects.none()
 
 
 class SearchTemplate(BaseSearchTemplate):
```

Comment:
change also this to `ListTemplateView`, add a docstring that explain that we can also search / filter